hypha/apply/dashboard/views.py

```diff
@@ -94,11 +97,26 @@ def get_context_data(self, **kwargs):
                 "rounds": self.rounds(),
                 "my_flagged": self.my_flagged(submissions),
                 "paf_waiting_for_assignment": self.paf_waiting_for_approver_assignment(),
+                "paf_for_review": self.paf_for_review(),
             }
         )
 
         return context
 
+    def paf_for_review(self):
+        if not self.request.user.is_apply_staff:
+            return {"count": None, "table": None}
+        project_settings = ProjectSettings.for_request(self.request)
+
+        paf_approvals = get_paf_for_review(
+            user=self.request.user, project_settings=project_settings
+        )
```

Comment:
In order to make this more testable, I would suggest keeping the project setting out of the picture from the util function. I would also suggest putting the function inside `services.py` instead of `utils.py`. If you think about it is not a common utility but a business function. 

```suggestion
        project_settings = ProjectSettings.for_request(self.request)

        paf_approvals = get_paf_for_review(
            user=self.request.user, is_paf_approval_sequential=project_settings.paf_approval_sequential
        )
```

Reply:
Alternatively, you can initialize the project settings inside the `get_paf_approvals` itself.